bigquery/indexer.py

```diff
@@ -318,29 +364,39 @@ def main():
     args = _parse_args()
     # Read dataset config files
     index_name = indexer_util.get_index_name(args.dataset_config_dir)
+    fields_index_name = '%s_fields' % index_name
     bigquery_config_path = os.path.join(args.dataset_config_dir,
                                         'bigquery.json')
     bigquery_config = indexer_util.parse_json_file(bigquery_config_path)
     deploy_config_path = os.path.join(args.dataset_config_dir, 'deploy.json')
     deploy_project_id = indexer_util.parse_json_file(
         deploy_config_path)['project_id']
-    es = indexer_util.maybe_create_elasticsearch_index(args.elasticsearch_url,
-                                                       index_name)
+    es = indexer_util.get_es_client(args.elasticsearch_url)
+    indexer_util.maybe_create_elasticsearch_index(
+        es, args.elasticsearch_url, index_name, shards=20)
```

Comment:
Is setting shards to 20 necessary? Can you remove all shard settings and see if it still works?

Reply:
This was kind of arbitrary, but I am pretty sure it won't work with a single shard (I'll test on Monday). Thinking about this more, I really think it should be configurable. For deploying to production we can scale the number of k8s data nodes (pods) which are running, in order to handle larger indices. Having the # shards < # nodes essentially results in nodes which are not utilized. I made this configurable via an env variable which defaults to the ES default (5). Let me know what you think.

https://www.elastic.co/guide/en/elasticsearch/guide/2.x/overallocation.html

Reply:
It worked locally with the default number of shards, so I'll remove it for now. If we need to add it back for k8s configuration I'll do it in that PR.